test-base/src/main/groovy/pl/allegro/tech/embeddedelasticsearch/EmbeddedElasticMultiIndexTypeBaseSpec.groovy

```diff
@@ -0,0 +1,29 @@
+package pl.allegro.tech.embeddedelasticsearch
+
+import static pl.allegro.tech.embeddedelasticsearch.SampleIndices.AMERICAN_PSYCHO
+import static pl.allegro.tech.embeddedelasticsearch.SampleIndices.AUDIO_BOOK_INDEX_TYPE
+import static pl.allegro.tech.embeddedelasticsearch.SampleIndices.BOOKS_INDEX_NAME
+import static pl.allegro.tech.embeddedelasticsearch.SampleIndices.PAPER_BOOK_INDEX_TYPE
+import static pl.allegro.tech.embeddedelasticsearch.SampleIndices.SHINING
+import static pl.allegro.tech.embeddedelasticsearch.SampleIndices.toJson
+
+abstract class EmbeddedElasticMultiIndexTypeBaseSpec extends EmbeddedElasticCoreApiBaseSpec {
```

Comment:
I would rather ignore selected spec in els 6.x tests than introduce another abstract class in hierarchy. Sth like this:

```
    @Ignore
    @Override
    def "should place document under right index and type"() {
        // not supported in ELS 6.x
    }
```

Reply:
In spock overriding test methods in subclasses isn't supported. In my opinion separated base classes per api version is cleaner. I changed the class names to be more readable :-)